Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -226,6 +227,7 @@ namespace AzToolsFramework
             void SetContainerEntity(AZ::Entity& entity);
 
             AZStd::unique_ptr<AZ::Entity> DetachContainerEntity();
+            bool DestroyContainerEntity();
```

Comment:
suggestion: Brief API comment about the return value (true deleted, false not) might be helpful